src/kixi/stats/distribution.cljc

```diff
@@ -646,6 +699,27 @@
   [{:keys [n alphas]}]
   (->DirichletMultinomial n alphas))
 
+(defn cauchy
+  "Returns a Cauchy distribution.
+  Params: {:location ∈ ℝ, :scale ∈ ℝ > 0}"
+  [{:keys [location scale]}]
+  (assert (pos? scale) (str "Scale (" scale ") must be positive"))
+  (->Cauchy location scale))
+
+(defn lognormal
+  "Returns a Lognormal distribution.
+  Params: {:location ∈ ℝ, :scale ∈ ℝ}"
+  [{:keys [location scale logmu logsd]}]
+  (->Lognormal (or location logmu) (or scale logsd)))
+
+(defn pareto
+  "Returns a Pareto distribution.
+  Params: {:scale ∈ ℝ > 0, :shape ∈ ℝ > 0}"
+  [{:keys [scale shape]}]
+  (assert (and (pos? scale) (pos? shape))
+          (str "Scale (" scale ") and shape (" shape "must be positive."))
```

Comment:
Unbalanced parens in the assert error string ( 😱

Reply:
Ah yes.  Thanks for catching that.